### ISSUE TITLE
DualListSelectorControl tooltips remain visible after VO blur

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly-react#7021]
**Describe the issue. What is the expected and unexpected behavior?**
The tooltips on the Dual List Selector component's controls do not currently get read by screen readers; whether this should be fixed since most likely the tooltip will match the `aria-label` of the button may require additional input. UPDATE: this portion of the issue looks to be resolved.

The tooltips also do not disappear immediately when navigating via VO (you have to navigate outside the button controls area it looks like, rather than simply navigating away from the button that has the tooltip currently displayed).

If the tooltips should be announced by screen readers, even if the contents of them matches the button's `aria-label`, then this might be similar to PR #6940 in that instead of using the Tooltip's `reference` prop, it should instead act as a wrapper around the DualListSelectorControl inner button element.

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**

Go to the PatternFly [Dual List Selector](https://www.patternfly.org/v4/components/dual-list-selector#basic-with-tooltips) page and use VO on the With Tooltips example. Notice that the buttons between the two option lists do not have tooltips announced by screen readers, nor do they immediately disappear after moving focus away from a button.

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
Could be considered a bug

**Comment by thatblindgeye** [comment]
Updates made in PR https://github.com/patternfly/patternfly-react/pull/7332 look to have resolve the issue of tooltips not getting announced. However, there is still a small issue where when navigating via VO controls from an active button to a disabled button, the tooltip remains visible. See video below:

https://user-images.githubusercontent.com/70952936/167696901-7564eb7e-603c-4252-afe9-d190e8fd8fd8.mov

**Comment by thatblindgeye** [comment]
This may be an issue with Voiceover/screen reader focus than anything to do with PF. Will keep this issue closed due to that and we can revisit if this is brought up again.
